rts/System/SpringApp.cpp

```diff
@@ -821,6 +822,8 @@ bool SpringApp::Update()
 	#else
 	// sic; Update can set the controller to null
 	retc = (        activeController == nullptr || activeController->Update());
+
+	auto lock = CLoadLock::GetScopedLock();
```

Comment:
How much cost does the lock incur at runtime (after load?).

Would it make sense to have a method to use after load without it?

Reply:
This:
https://github.com/beyond-all-reason/spring/blob/d2cfb9f7a96f2e5faf0b3b9bae7f9c60d5e0ff5f/rts/System/LoadLock.h#L12-L13
every time you lock (create ScopedLock)

and that:
https://github.com/beyond-all-reason/spring/blob/d2cfb9f7a96f2e5faf0b3b9bae7f9c60d5e0ff5f/rts/System/LoadLock.h#L19-L20
every time you unlock (the ScopedLock object gets destroyed)